pkgs/development/compilers/edk2/default.nix

```diff
@@ -72,7 +89,7 @@ edk2 = stdenv.mkDerivation {
         sed -i Conf/tools_def.txt \
           -e 's|DEFINE GCC48_IA32_PREFIX       = /usr/bin/|DEFINE GCC48_IA32_PREFIX       = ""|' \
           -e 's|DEFINE GCC48_X64_PREFIX        = /usr/bin/|DEFINE GCC48_X64_PREFIX        = ""|' \
```

Comment:
N.B. this was GCC49 above.

Reply:
And is defined above already like so:

```
           -e 's|MYTOOLS|GCC49|' \
```

This smells of code that now does nothing.